db/seeds.rb

```diff
@@ -9,13 +9,13 @@
 comedies = Category.create(name: "Comedies")
 action = Category.create(name: "Action")
 dramas = Category.create(name: "Dramas")
-Video.create(title: "Southpark", description: "Colorado town of crazy kids", category: comedies,
+southpark = Video.create(title: "Southpark", description: "Colorado town of crazy kids", category: comedies,
```

Comment:
`.create!` actually throws an exception if the record created is not valid - so it does not silently ignore it.